international-journal-of-learner-corpus-research.csl

```diff
@@ -0,0 +1,1514 @@
+              <if variable="container-title" match="any">
+                <!--Only print year for cases published in reporters-->
+                <date variable="issued" form="numeric" date-parts="year"/>
+              </if>
+              <else>
+                <date variable="issued" form="text"/>
+              </else>
+            </choose>
+          </group>
+        </group>
+      </if>
+      <else-if type="bill legislation" match="any">
+        <group prefix=", " delimiter=" ">
+          <group delimiter=", ">
+            <choose>
+              <if variable="number">
+                <!--There's a public law number-->
+                <text variable="number" prefix="Pub. L. No. "/>
+                <group delimiter=" ">
+                  <!--change to label variable="section" as that becomes available -->
+                  <text term="section" form="symbol"/>
+                  <text variable="section"/>
+                </group>
+                <group delimiter=" ">
+                  <text variable="volume"/>
+                  <text variable="container-title"/>
+                  <text variable="page-first"/>
+                </group>
+              </if>
+              <else>
+                <group delimiter=" ">
+                  <text variable="volume"/>
+                  <text variable="container-title"/>
+                  <!--change to label variable="section" as that becomes available -->
+                  <text term="section" form="symbol"/>
+                  <text variable="section"/>
+                </group>
+              </else>
+            </choose>
+          </group>
+          <date variable="issued" prefix="(" suffix=")">
+            <date-part name="year"/>
+          </date>
+        </group>
+      </else-if>
+      <else-if type="treaty">
+        <group delimiter=" ">
+          <number variable="volume"/>
+          <text variable="container-title"/>
+          <text variable="page"/>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <!-- As per the IJLCR stylesheet, page numbers are not proceeded by p. or pp. in in-text citations. The following macro was therefore added. -->
+  <macro name="point-locators">
+    <group>
+      <choose>
+        <if locator="page" match="none">
+          <label variable="locator" form="short" suffix=" "/>
+        </if>
+      </choose>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <!-- The IJLCR stylesheet actually states: "When citing or quoting a publication with three or more authors for the first time in the text, please list all authors’ names. For every consecutive mention of this reference, please give only the first author plus “et al." As far as I know, this cannot be achieved in Zotero. Et al. is therefore always printed when there are 3 or more authors -->
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
```

Comment:
```suggestion
  <citation et-al-min="6" et-al-use-first="1" et-al-subsequent-min="3" et-al-subsequent-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" collapse="year" givenname-disambiguation-rule="primary-name">

```

Reply:
This is taken straight from APA 6th -- not sure why that's not in your version of the style? You definitely want the et-al parts, see if you also want the additional attributes at the end.

Reply:
I've just tested for suggestion and it looks great, thank you!